__tests__/newsApi.test.js

```diff
@@ -50,3 +50,21 @@ describe('Core Task 2: GET /api/topics', () => {
     });
 });
 
+describe('Core Task 3: GET request /api', () => {
+    test('Status 200: An object describing all the available endpoints on this API - extensive use of jest Extender functionality', () => {
+        return request(app)
+               .get('/api')
+               .expect(200)
+               .then(({body})=>{
+                const endpointInformation = body.endpoints
+                expect(typeof endpointInformation).toBe('object')
```

Comment:
This is good - but will become very long as you add more endpoints! how could you make this a bit less repetitive? Consider iterating through your result and using something like .toMatchObject - you can find out about it [in the jest docs](https://jestjs.io/docs/expect#tomatchobjectobject)